### ISSUE TITLE
PineAPPL needs a logo

### ISSUE THREAD

**Issue by cschwan** [N3PDF/pineappl#21]

**Comment by AleCandido** [comment]
I was looking to the issues and I noticed this one-year-old one.

I would suggest something like this for the concept:
![pineappl](https://user-images.githubusercontent.com/10830876/125981497-8576789b-060a-42f7-8a67-63970528b55b.png)
(the svg is not supported by GitHub and thus attached [pineappl.zip](https://github.com/N3PDF/pineappl/files/6832251/pineappl.zip), with an alternative layout)

The font is random, colors are almost random, and even the layout: the idea I wanted to propose was just the gear with the slips on top :)
(if you wish the slips are taken from [here](https://misskyliedesign.com/wp-content/uploads/2020/06/pineapple-monogram-svg.jpg) and there is also the full pineapple outline)

**Comment by cschwan** [comment]
Thanks for looking into that. The first pineapple from you last link actually quite nice, I like that the body looks like a grid!

**Comment by AleCandido** [comment]
That's another chance of course :)

**Comment by felixhekhorn** [comment]
If only PineAPPL had a logo so we could put that in talks ...

**Comment by AleCandido** [comment]
I guess he was referring to DIS, in which he's going to present EKO and yadism, and thus he will mention the combination (so PineAPPL).

**Comment by cschwan** [comment]
Yes, having a Logo would be useful ...

**Comment by felixhekhorn** [comment]
> I guess he was referring to DIS

Indeed. Now, until then nothing will happen - after that let me point out what I already said: https://github.com/N3PDF/pineappl/issues/21#issuecomment-939104322

**Comment by felixhekhorn** [comment]
just saying that @AleCandido will give another talk without a logo and I, myself, planning to give a dedicated talk about the theory pipeline (and so pineappl at it's center) at the EIC meeting at the beginning of August ...

practical suggestions: why don't we accept @AleCandido suggestion [above](https://github.com/N3PDF/pineappl/issues/21#issuecomment-881590989) ? :upside_down_face:

**Comment by cschwan** [comment]
Alessandro's suggestion is a bit too generic to me, let me work on a logo. For the time being I think we should draw the attention to the updated website, what do you think?

**Comment by AleCandido** [comment]
The former suggestion is rather old, and not my favorite.

I'd like to make at least a try also with a Pine, rather than a Pineapple. I'm still a bit affectionate to the idea of the theory **pine**line.

**Comment by felixhekhorn** [comment]
@cschwan any progress on this by chance?

**Comment by felixhekhorn** [comment]
I need to prevent this issue to become stale ... and keep hoping ...

**Comment by scarlehoff** [comment]
What should I use for the pineappl logo for presentations?

**Comment by AleCandido** [comment]
No official logo yet, last time I used something like this
![image](https://user-images.githubusercontent.com/10830876/201545696-2ca4f3e2-7b58-4b5d-8483-2b05e2d3fa50.png)

**Comment by AleCandido** [comment]
Just to mention: I was working on another candidate logo for PineAPPL, but now I don't have time to complete anything.

However, the alternative subject (other than a pineapple as above) was a pine, and I was considering to start from something like this:
![istockphoto-499771256-612x612](https://user-images.githubusercontent.com/10830876/201545811-beb8898f-5127-4b5e-9450-ffaf31de2502.jpg)
![stone-pine](https://user-images.githubusercontent.com/10830876/201545815-2597e392-225c-4ec6-b3f7-897a919058a9.gif)

**Comment by Radonirinaunimi** [comment]
Here are some new propositions for the logo:
- [pineappl-logo-grey-1.pdf](https://github.com/user-attachments/files/20638679/pineappl-logo-grey-1.pdf)
- [pineappl-logo-grey-2.pdf](https://github.com/user-attachments/files/20638680/pineappl-logo-grey-2.pdf)
- [pineappl-logo-grid.pdf](https://github.com/user-attachments/files/20638682/pineappl-logo-grid.pdf)
- [pineappl-logo-orange-1.pdf](https://github.com/user-attachments/files/20638683/pineappl-logo-orange-1.pdf)
- [pineappl-logo-orange-2.pdf](https://github.com/user-attachments/files/20638684/pineappl-logo-orange-2.pdf)

The source `SVG` files are available inside [sources.tar.gz](https://github.com/user-attachments/files/20638687/sources.tar.gz).

**Comment by Radonirinaunimi** [comment]
Or, actually, a more colored version ([pineappl-logo-grid-orange.svg.zip](https://github.com/user-attachments/files/20639935/pineappl-logo-grid-orange.svg.zip)):

<img width="1159" alt="Image" src="https://github.com/user-attachments/assets/47a4c0fe-5d1c-4033-b135-bab6dba9c8a0" />

**Comment by cschwan** [comment]
> Or, actually, a more coloured version ([pineappl-logo-grid-orange.svg.zip](https://github.com/user-attachments/files/20639935/pineappl-logo-grid-orange.svg.zip)):

I also like this one a lot! However, could you change the font to 'DejaVu Sans Mono', maybe even bold, and change the text to the official capitalization: PineAPPL.

**Comment by Radonirinaunimi** [comment]
Yes, here it is ([pineappl-logo-grid-orange-2.svg.zip](https://github.com/user-attachments/files/20644098/pineappl-logo-grid-orange-2.svg.zip)):

<img width="906" alt="Image" src="https://github.com/user-attachments/assets/40ad90f0-9c65-43aa-95f4-a751edefb0ae" />

**Comment by cschwan** [comment]
Can you
- put the bottom part of the green section a bit lower so there's no gap to the grid?
- the ratio of the pineapple seems a bit off, can you maybe add another column to the grid?
- I'm not quite happy with the font choice, maybe you'll find a better one.

**Comment by Radonirinaunimi** [comment]
Got it! As for the font, is there maybe that you like from here https://wordmark.it/ (you can enter "PineAPPL" in the field and it'll show you it in the different font)?
